frontend/simple/js/backend/hapi.js

```diff
@@ -85,17 +85,21 @@ export class HapiBackend extends Backend {
     if (this.subscriptions().indexOf(contractId) === -1) {
       return console.error('HapiBackend.unsubscribe: not subscribed!', contractId)
     }
-    var res = await primus.unsub(contractId)
-    store.commit('removeContract', contractId)
+    let res = await primus.unsub(contractId)
```

Comment:
well you shouldn't necessarily replace lets with const everywhere arbitrarily because the `let`'s  are block scoped and `const` are scoped local to the function just like `var`. In some instances that may matter to garbage collection but its  a trivial difference.